e2e/_suites/fleet/ingest_manager_test.go

```diff
@@ -132,6 +136,24 @@ func InitializeIngestManagerTestSuite(ctx *godog.TestSuiteContext) {
 			docker.PullImages(images)
 		}
 
+		if !shell.GetEnvBool("SKIP_FETCH") {
```

Comment:
I think Fleet test suite does not need the metricbeat compose files. Do you think this applies to the metricbeat test suite instead?